ccdproc/core.py

```diff
@@ -703,6 +703,7 @@ def gain_correct(ccd, gain, gain_unit=None):
         gain_value = gain
 
     result = ccd.multiply(gain_value)
+    result.meta = ccd.meta.copy()
```

Comment:
But I don't know if it's worth keeping the meta only if and only if the other operand has none. That's inconsistent with the case that the other operand is a CCDData, then the meta is always discarded.